### ISSUE TITLE
Build linux binaries with Travis?

### ISSUE THREAD

**Issue by sjakobi** [HeinrichApfelmus/hyper-haskell#7]
I've only heard of this but don't have any experience with it myself.

It's documented here: https://docs.travis-ci.com/user/deployment/releases

**Comment by HeinrichApfelmus** [comment]
Thanks for the suggestion, I will look into this. One thing I don't understand is whether I have to make an authentication token part of the source code (.travis.yml). That could be insecure.

**Comment by pgrosslicht** [comment]
No, if you do want to include some kind of secret like an authentication token, you can encrypt it and use the encrypted token instead. For more information on how to use that, look [here](https://docs.travis-ci.com/user/encryption-keys/).
